queries.sql

```diff
@@ -1,4 +1,8 @@
 UPDATE teams 
 SET abbreviation = WAS
   WHERE abbreviation
-IS WHS;
\ No newline at end of file
+IS WHS;
```

Comment:
`IS` only works with `NULL`. 

`UPDATE teams SET abbreviation = 'WAS' WHERE abbreviation = 'WSH';`